mappings/net/minecraft/block/enums/Half.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS bql
+CLASS bql net/minecraft/block/enums/Half
```

Comment:
Half or a more specific Half?

Reply:
It's just a generic top/bottom half of a block. I was initially going to name this calls BlockHalf, however another class has that name. The other class is used for describing blocks that exist as two separate blocks, like doors and tall grass. The class I have named is used to describe the half of a specific block, like trap doors and stairs.

Reply:
Then both should probably be renamed. Having both BlockHalf and Half is confusing if you don't remember which one is which.

Reply:
Okay, how do these names sound?

BlockHalf -> DoubleBlockHalf
Half -> BlockHalf

Reply:
better than any alternatives I've heard. Do double plants use the former as well?

Reply:
@Prospector The "TallPlantBlock" class uses it, I think it's all the same class there.